hieradata/class/backend.yaml

```diff
@@ -33,6 +33,12 @@ govuk::apps::share_sale_publisher::mongodb_nodes:
   - 'mongo-2.backend'
   - 'mongo-3.backend'
 
+govuk::apps::specialist_publisher::mongodb_name: 'specialist_publisher_production'
```

Comment:
You possibly don't have access to this repo, it shows that specialist publisher uses a common mongo db.